### PR TITLE
feat(dev): add new event page loaded

### PR DESCRIPTION
**Pull request by ricardoapaes**
Adding a new event when the page is loaded and you can interact with the DOM using the puppeter, below is a practical example:

> Adding a new div to the html for example:
> - Request login to your internal system before starting whatsapp.

### CODE REVIEW

**Comment by PurpShell** [comment]
hey! i wouldn't get how this is effective? Does this allow editing of the UI? Maybe something we can accomplish differently. 

Thank you for your contributions. 

Best regards, 

[PurpShell](https://github.com/PurpShell)

**Comment by JoseHM8A** [comment]
wow, i think this solved my problem getting the profile pictures of participants/groups.

Thank so much

**Comment by ricardoapaes** [comment]
> hey! i wouldn't get how this is effective? Does this allow editing of the UI? Maybe something we can accomplish differently.
> 
> Thank you for your contributions.
> 
> Best regards,
> 
> [PurpShell](https://github.com/PurpShell)

@PurpShell 

Since I'm using the puppeteer in "headless: false" mode and I'm integrating with my web software, I'd like before opening whatsapp itself the user would login to my software so that I can get session information to be used later, so I add other html tags that contain the login form.

I couldn't find an easier way to do this since the events provided by the platform are only emitted after whatsapp is loaded.

If you have a better suggestion for my problem I am open to other opinions.

**Comment by PurpShell** [comment]
> > hey! i wouldn't get how this is effective? Does this allow editing of the UI? Maybe something we can accomplish differently.
> > Thank you for your contributions.
> > Best regards,
> > [PurpShell](https://github.com/PurpShell)
> 
> @PurpShell
> 
> Since I'm using the puppeteer in "headless: false" mode and I'm integrating with my web software, I'd like before opening whatsapp itself the user would login to my software so that I can get session information to be used later, so I add other html tags that contain the login form.
> 
> I couldn't find an easier way to do this since the events provided by the platform are only emitted after whatsapp is loaded.
> 
> If you have a better suggestion for my problem I am open to other opinions.

looks good.

**Comment by PurpShell** [comment]
unstale

File: src/Client.js [hunk 1]
```diff
@@ -101,6 +101,8 @@ class Client extends EventEmitter {
             timeout: 0,
         });
 
+        this.emit(Events.PAGE_LOADED, page);
```
Comment (PurpShell) [inline src/Client.js:104]:
```suggestion
        await page.waitForSelector('[class=web]');
        this.emit(Events.PAGE_LOADED, page);
```

**Review by purpshell — CHANGES_REQUESTED** [review]
just a slight change

**Pull request closed without merging**